lib/tasks/data-key.rake

```diff
@@ -8,4 +8,63 @@ namespace :"data-key" do
     key = Base64.strict_encode64(Slosilo::Symmetric.new.random_key)
     print key
   end
+
+  desc "Validates the value of ENV['CONJUR_DATA_KEY']"
+  # This task validate the master data key that is stored in `ENV['CONJUR_DATA_KEY']`
+  # The task runs in two modes:
+  # First mode:
+  # ==========
+  #   Once the database is created, a role + resource + secret will be created using the master key.
+  #
+  # Second mode:
+  # ===========
+  #   Each time the server restarts this task will attempt to retrieve the
+  #   secret, if the provided key does not match the original master the secret retrieval
+  #   will fail and the task will exit with error.
+  #
+  # Side Note
+  # =========
+  #  What's the 'environment' task in Rake?
+  #  the :validate => :environment means "do environment before validate".
+  #  You can get access to your models, and in fact, your whole environment
+  #  by making tasks dependent on the environment task.
+  task :validate => [:environment] do
+    data_key = ENV['CONJUR_DATA_KEY']
+
+    if data_key.nil? || data_key.empty?
+      raise Errors::System::EmptyConjurDataKey
+      exit 1
```

Comment:
will we get to this line after we raised an error?

Reply:
When I tested it the process exited with error.

Reply:
This `exit` is unreachable.  I'm guessing the process exited with an error because of the `raise`, not because of line 36, which you can remove.